env0/resource_configuration_variable.go

```diff
@@ -68,6 +70,13 @@ func resourceConfigurationVariable() *schema.Resource {
 				ConflictsWith: []string{"template_id", "project_id", "is_required", "is_read_only"},
 				ForceNew:      true,
 			},
+			"sub_environment_alias": {
+				Type:         schema.TypeString,
+				Description:  "create the variable for sub templates of a workflow. Requires template_id to be set.",
+				Optional:     true,
+				ForceNew:     true,
+				RequiredWith: []string{"template_id"},
+			},
```

Comment:
commenting here, but in the data file you changed the description of `template_id` to say it is required when using an alias, let's have it also here